database/api/routes.md

```diff
@@ -30,26 +30,27 @@ NOTE:: recon json key has been removed from previous versions
             lastModified: String
         }
     ]
+    NOTE: Will these be sorted by program_block_id? Can we gurantee that a inProgramBlock has been created first?
     programBlock: [
         {
-            id: Integer
+            programBlockId: Integer
             inProgramBlock: Integer or NULL
             name: String
             qualifiedName: String
         }
     ]
     data: [
         {
-            id: Integer
+            dataId: Integer
             inProgramBlock: Integer or NULL
             name: String
             qualifiedName: String
         }
     ]
     port: [
         {
-            id: Integer
-            onProgramBlock: Integer
+            portId: Integer
+            inProgramBlock: Integer
```

Comment:
i don't believe this was not a typo in the original db

Reply:
Oops thought it was, will fix

Reply:
can you change it back in the spec please?